.pre-commit-config.yaml

```diff
@@ -33,6 +33,11 @@ repos:
     hooks:
       - id: prettier
         types_or: [ini, json, toml, yaml]
+  - repo: https://github.com/scop/pre-commit-shfmt
+    rev: v3.8.0-1
+    hooks:
+      - id: shfmt
+        args: ["-i", "2"]
```

Comment:
I would think you want `--diff` as well. according to the [docs](https://github.com/mvdan/sh/blob/master/cmd/shfmt/shfmt.1.scd#generic-flags):

```
-d, --diff Error with a diff when the formatting differs.
```

Reply:
And `--write` apparently as well.  Thanks.  d464fb3781a2b688517cc5148d4b52dfc0aa95bc